src/io/libxdrf.c

```diff
@@ -0,0 +1,1008 @@
+/*
```

Comment:
I'd be happy to, but is there any such library that is accessible in a user-friendly manner? (as in: obtainable without installing the whole lot of gromacs, mdanalysis or vmd?)